src/producer/index.js

```diff
@@ -14,6 +14,66 @@ module.exports = ({
   const sendMessages = createSendMessages({ cluster, partitioner })
   const logger = rootLogger.namespace('Producer')
 
+  const sendBatch = async ({ acks, timeout, compression, topicMessages }) => {
+    if (topicMessages.some(({ topic }) => !topic)) {
```

Comment:
Let's validate that `topicMessages` is not empty

Reply:
c231143